config.json

```diff
@@ -424,7 +369,7 @@
         }
       },
       "defaultValues":{
-        "toolName": "terminal",
+        "toolName": "script",
```

Comment:
Why did this name change? I kind of prefer terminal as it is more clear what the entry point exe is as opposed to script.